test/controllers/concerns/authentication_test.rb

```diff
@@ -223,122 +284,15 @@ class AuthenticationTest < ActionDispatch::IntegrationTest
           }
         }.to_json
       )
+      AuthenticatedMockController.any_instance
+                                 .stubs(:valid_cert_endpoint?)
+                                 .returns(true)
       HostInventoryApi.any_instance.expects(:hosts)
                       .raises(Faraday::Error.new(''))
       RbacApi.expects(:new).never
-      get profiles_url, headers: { 'X-RH-IDENTITY': encoded_header }
-      assert_response :forbidden
-    end
-
-    should 'allow access to profiles#index' do
```

Comment:
Please do not remove these tests. They test cert auth

Reply:
They've been moved to profile_controller_test.

Reply:
Ah! sorry, I didn't realize that. They're all related to cert auth and are not specific to any one controller, though. IMO they belong in `authentication_test` since that's the code they're meant to test.

Reply:
These are specific for the profiles controller, to test the integration of authentication within that controller, hence the move.